src/core/utilities.js

```diff
@@ -113,11 +111,12 @@ function is( type, obj ) {
 }
 
 var getUrlParams = function() {
-	var i, current,
-		location = window.location || { search: "", protocol: "file:" },
-		params = location.search.slice( 1 ).split( "&" ),
-		length = params.length,
-		urlParams = {};
+	var i, current, location, params, length;
+	var urlParams = {};
+
+	location = window.location;
+	params = location.search.slice( 1 ).split( "&" );
+	length = params.length;
```

Comment:
Also applies to these two, as long as the order stays the same. Just stop using onevar...